uiowa/uiowa
# IE Overflow Fixes for Content Teasers

Resolves #1483 

## To Test in IE 11
Go to https://founiversity-shared-services.dev.drupal.uiowa.edu/uss-contacts-strategic-communications and make sure the person images aren't overflowing and gigantic
<img width="609" alt="Screen Shot 2020-06-01 at 3 46 06 PM" src="https://user-images.githubusercontent.com/4663676/83452811-07cc6a80-a41f-11ea-94a9-dba4f0669cb5.png">

Find a news paragraph grid make sure the headlines aren't overflowing.
<img width="1559" alt="Screen Shot 2020-06-01 at 3 22 00 PM" src="https://user-images.githubusercontent.com/4663676/83452827-0f8c0f00-a41f-11ea-9e28-4cb582f0ee92.png">
